examples/kubernetes/static_provisioning/README.md

```diff
@@ -4,10 +4,11 @@ This example shows how to make a static provisioned Mountpoint for S3 persistent
 ## Examples in this folder
 - `static_provisioning.yaml` - spawning a pod which creates a file with name as the current date/time
 - `non_root.yaml` - same as above, but the pod is spawned as non-root (uid `1000`, gid `2000`)
+- `s3_express_specify_az.yaml` - same as above, but this uses [S3 Express](https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-express-one-zone.html) bucket and shows how to specify the availability zone (AZ) of the [pod assignment](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity)
```

Comment:
IMO it's better to be explicit and use the full "S3 Express One Zone" name in this doc even if it is a bit long.

Reply:
yeah, it's always "S3 Express One Zone"

Reply:
```suggestion
- `s3_express_specify_az.yaml` - same as above, but this uses a [S3 Express One Zone](https://docs.aws.amazon.com/AmazonS3/latest/userguide/s3-express-one-zone.html) directory bucket and shows how to specify the availability zone (AZ) of the [pod assignment](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity) to co-locate the pod with the bucket for lower latency access
```